print Tables

```diff
@@ -0,0 +1,7 @@
+# Python Program to print Tables of user given number till that number.
```

Comment:
Please make separate files for ques and solution with code extension.py in solution file. Follow the folder structure.